File: app/src/main/java/com/example/outsidehacks17/outsidehacks/MainActivity.java
package com.example.outsidehacks17.outsidehacks;

import android.content.Intent;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        Intent loginScreen = new Intent(MainActivity.this, LoginScreen.class);
        loginScreen.putExtra("UserFactory", new UserFactory());
        Intent artistScreen = new Intent(MainActivity.this, ArtistsActivity.class);
       // startActivity(artistScreen);
        startActivity(loginScreen);
        finish();
    }
}

File: app/src/main/java/meetoutside/meetoutsidev2/MainActivity.kt
package meetoutside.meetoutsidev2

import android.content.Intent
import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import com.google.firebase.database.DatabaseReference
import com.google.firebase.database.FirebaseDatabase



class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val database: FirebaseDatabase = FirebaseDatabase.getInstance()
        val myRef: DatabaseReference = database.getReference("message")

        myRef.setValue("Hello, World!")
        startActivity(Intent(this@MainActivity, SignInActivity::class.java))
        finish()
    }
}
